src/app/storage.js

```diff
@@ -0,0 +1,97 @@
+var utils = require('./utils');
+
+function Storage (updateFiles) {
+
+  var EDITOR_SIZE_CACHE_KEY = 'editor-size-cache';
+
+  this.rename = function (originalName, newName) {
+    var content = this.get(originalName);
+    this.set(newName, content);
+    this.remove(originalName);
+  };
+
+  this.remove = function (name) {
+    window.localStorage.removeItem(name);
+  };
+
+  this.setEditorSize = function (size) {
```

Comment:
`storage.get()` and `storage.set()` are public so it could be safely called from within the editor?

What it is used for? I don't see any references - though github diffs lately are weird and don't show everything.